src/mobile/src/main/java/tupperdate/android/App.kt

```diff
@@ -79,6 +85,11 @@ fun TupperdateApp(
         }
 
         is UiState.LoggedIn -> {
+            val scope = LifecycleOwnerAmbient.current.lifecycleScope
+            scope.launch {
+                api.users.updateProfile()
+            }
```

Comment:
You want to use a `LaunchedEffect` for this

Reply:
I will check it out and add an issue, thanks